src/components/Table/Budget/BudgetLineItemTable/BudgetLineItemTableView.tsx

```diff
@@ -152,7 +152,7 @@ function BudgetLineItemTableView({
 					<Box my={2} display="flex" flexWrap="wrap">
 						{Object.entries(filterList).map((element) => {
 							if (element[1] && Array.isArray(element[1])) {
```

Comment:
what is `1`  `0` ?

Reply:
element is a array, 1 is the value and 0 is the key of the object

Reply:
Yeah it is obvious. My whole point is about keeping meaningful names so that if another developer works on this file , they won't have to put a lot of time just to understand the code